packages/storefront-sdk/src/client/index.ts

```diff
@@ -12,8 +23,16 @@ export class StorefrontClient {
 		});
 	}
 
-	// NEW METHODS GO HERE :)
-	placeholder() {
-		return this.#graphqlClient.query('', {}).toPromise();
+	query<QData = any, QVariables extends AnyVariables = any>({
```

Comment:
We can tell ESLint to chill with this:

![eslint Unexpected any  Specify a different type](https://user-images.githubusercontent.com/5732000/210670122-a3f6b236-47c3-489c-b9f8-ce74c8a8eff0.png)


```suggestion
	// eslint-disable-next-line @typescript-eslint/no-explicit-any
	query<QData = any, QVariables extends AnyVariables = any>({
```